From 7503606f714133987231a43c450d4481968c9e38 Mon Sep 17 00:00:00 2001
From: rebecca skinner <rebecca@rebeccaskinner.net>
Date: Sun, 6 Jun 2021 13:08:33 -0500
Subject: [PATCH] fix deps

---
 hakyll.cabal | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hakyll.cabal b/hakyll.cabal
index c582934..bfbe347 100644
--- a/hakyll.cabal
+++ b/hakyll.cabal
@@ -175,7 +175,7 @@ Library
     blaze-markup         >= 0.5.1    && < 0.9,
     bytestring           >= 0.9      && < 0.11,
     containers           >= 0.3      && < 0.7,
-    cryptonite           >= 0.25     && < 0.28,
+    cryptonite           >= 0.25     && < 0.29,
     data-default         >= 0.4      && < 0.8,
     deepseq              >= 1.3      && < 1.5,
     directory            >= 1.2.7.0  && < 1.4,
@@ -185,7 +185,7 @@ Library
     memory               >= 0.14.18  && < 0.16,
     mtl                  >= 1        && < 2.3,
     network-uri          >= 2.6      && < 2.7,
-    optparse-applicative >= 0.12     && < 0.16,
+    optparse-applicative >= 0.12     && < 0.17,
     parsec               >= 3.0      && < 3.2,
     process              >= 1.6      && < 1.7,
     random               >= 1.0      && < 1.3,
@@ -237,7 +237,7 @@ Library
     Other-Modules:
       Hakyll.Web.Pandoc.Binary
     Build-Depends:
-      pandoc >= 2.11 && < 2.12
+      pandoc >= 2.13 && < 2.14
     Cpp-options:
       -DUSE_PANDOC
 
@@ -270,7 +270,7 @@ Test-suite hakyll-tests
   Build-Depends:
     hakyll,
     QuickCheck                 >= 2.8  && < 2.15,
-    tasty                      >= 0.11 && < 1.4,
+    tasty                      >= 0.11 && <= 1.4.1,
     tasty-golden               >= 2.3  && < 2.4,
     tasty-hunit                >= 0.9  && < 0.11,
     tasty-quickcheck           >= 0.8  && < 0.11,
-- 
2.31.1

